docs/conf.py

```diff
@@ -94,8 +95,16 @@
 #
 # The full version, including alpha/beta/rc tags.
 release = version("discretize")
+discretize_version = parse(release)
+
+# The short X.Y version.
+version = discretize_version.public
+if discretize_version.is_devrelease:
+    branch = "main"
+else:
+    branch = f"v{version}"
```

Comment:
We should also remove the leading `v` from here for the version to be parsable.
```suggestion
    branch = f"{version}"
```